frontend/src/components/elements/ExceptionElement/ExceptionElement.scss

```diff
@@ -14,11 +14,22 @@
  * limitations under the License.
  */
 
+@import "src/assets/css/variables";
+
 .reportview-container {
   .exception .message {
     margin-bottom: 0.5rem;
   }
 
+  .exception .message .type {
+    font-size: larger;
+    font-weight: bold;
+  }
+
+  .exception .message code {
+    color: $primary;
+  }
```

Comment:
I think this color and the font-size above look funny.

Reply:
I think the green-on-red kinda looks like a tacky Christmas tree:

![image](https://user-images.githubusercontent.com/709022/67800110-fa43a200-fa43-11e9-9f63-0306298b5df2.png)

vs

![image](https://user-images.githubusercontent.com/709022/67800149-0891be00-fa44-11e9-8163-e3c2b1e1a56e.png)

(But I'll back out the change!)